### PR TITLE
test pizza App functionalities need to be added  

### PR DESCRIPTION
**Pull request by Dztester2013**
create few contractors methods  and few methods for test

### CODE REVIEW

File: src/Pizza/AppTest.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+String model;
+
+    public AppTest(int var, int val, String vendor, String model) {
+        this.var = var;
+        this.val = val;
+        this.vendor = vendor;
+        this.model = model;
+        System.out.println("AppTest constructor with 4 parameters ");
+    }
+
```
Comment (mhshahib) [inline src/Pizza/AppTest.java:20]: please remove empty line

File: src/Pizza/AppTest.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        this.var = var;
+        this.val = val;
+        this.vendor = vendor;
+        this.model = model;
+        System.out.println("AppTest constructor  with 1 parameters ");
+    }
+
+    public AppTest() {
+        System.out.println("Empty  constructor no parameters  ");
+
```
Comment (mhshahib) [inline src/Pizza/AppTest.java:41]: remove empty space inside of a method or constructor

File: src/Pizza/AppTest.java [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    }
+
+    public static String  compter (String model){
```
Comment (mhshahib) [inline src/Pizza/AppTest.java:44]: method name should be meaningful and please update method name

**Review by mhshahib — CHANGES_REQUESTED** [review]
please follow all the comments and push updated code.

File: src/Pizza/AppTest.java (hunk 1 above)
Comment (Dztester2013) [reply src/Pizza/AppTest.java:20]: empty line removed  based in the comment

File: src/Pizza/AppTest.java (hunk 2 above)
Comment (Dztester2013) [reply src/Pizza/AppTest.java]: all empty space inside of methods and constructors deleted

File: src/Pizza/AppTest.java (hunk 3 above)
Comment (Dztester2013) [reply src/Pizza/AppTest.java]: method name updated

**Review by mhshahib — APPROVED** [review]
now code is fine and you can merge your code after resolve all the thread/ comments.

**Comment by Dztester2013** [comment]
code is merge with main branch and delete Amar_Alili branch

**Pull request merged**